cd /media/pi/FLASH_DRIVE/pi_files &&  python3 Auto_setup.py && python3 main.py > error.log && sudo reboot
